portal/config.py

```diff
@@ -110,6 +110,10 @@ class BaseConfig(object):
     SMARTLING_USER_SECRET = os.environ.get('SMARTLING_USER_SECRET', None)
     SMARTLING_PROJECT_ID = os.environ.get('SMARTLING_PROJECT_ID', None)
 
+    RECAPTCHA_ENABLED = True
+    RECAPTCHA_SITE_KEY = "6LcNNigUAAAAAIgA9i1ZUeGzDf2dJg5ghMUHVggR"
+    RECAPTCHA_SECRET_KEY = "6LcNNigUAAAAAFp84MbFhVMj64SmcfhQZpFRk8ai"
```

Comment:
Not required to be site-specific; we just need to add the relevant domains to the whitelist in the [google recaptcha admin panel](https://www.google.com/recaptcha/admin) (which should already be done).

We COULD use different site keys if we wanted to keep track of verification attempts in the admin panel on a per-site basis. But for the initial implementation I don't think that's necessary.